core/src/empty/empty1.svg

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This is just an SVG that wraps a bitmap image :)) we want a truly vector-based SVG. But anyway, I'll make a PR to fix this for you to review